### PR TITLE
Needed a useful error. Pardon the bad go

### PR DESCRIPTION
**Pull request by pburkholder**
We'll probably want to reject this for a standard approach to emitting useful error message. Anyhow my first PR in GoLang (gotta start someplace).

### CODE REVIEW

File: models/opencontrol.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ func (openControl *OpenControl) LoadComponents(directory string) []error {
 	var wg sync.WaitGroup
 	componentsDir, err := ioutil.ReadDir(directory)
 	if err != nil {
-		return []error{ErrReadDir}
+		return []error{errors.New("Error: Unable to read the directory "+directory)}
```
Comment (afeld) [inline models/opencontrol.go]: We might be able to simply do

```go
return []error{err}
```

I'd do that for the other occurrences of `ErrReadDir` too, and get rid of that error completely for now.

**Comment by afeld** [comment]
> We'll probably want to reject this for a standard approach to emitting useful error message.

The problem in this case is that we were being too clever in trying to pass a custom error message rather than just surfacing the pre-existing error. I think that's fine for now, until we want to get really crafty with the error messages.

> my first PR in GoLang

🎉

**Comment by afeld** [comment]
Also, if you're feeling bold, mind adding a test to ensure that the directory is reported? It will go into `models/opencontrol_test.go`, and can look like a variation of [this](https://github.com/opencontrol/compliance-masonry/blob/9b873de27bf726fb4d15fff19cafb281e9222664/models/opencontrol_test.go#L58-L63), but checking the returned `error`.

File: models/opencontrol.go (hunk 1 above)
Comment (pburkholder) [inline models/opencontrol.go]: I forgot about this while I was writing tests. It works with the string, now, but now that I have tests working we could go this route too.

**Comment by pburkholder** [comment]
ErrReadDir is gone and tests are in place. Good to go?

File: models/opencontrol.go (hunk 1 above)
Comment (afeld) [inline models/opencontrol.go]: We might need a more holistic approach, but I'm good with this for now.

**Comment by afeld** [comment]
Thanks so much!

**Pull request merged**